src/search/indexing/local_import.rs

```diff
@@ -29,12 +28,11 @@ pub async fn index_local(pool: PgPool) -> Result<Vec<UploadSearchMod>, IndexingE
                 ",
                 result.id
             )
-            .fetch_many(&pool)
-            .try_filter_map(|e| async { Ok(e.right().map(|c| c.version)) })
-            .try_collect::<Vec<String>>()
-            .await?;
+                .fetch_many(&pool)
+                .try_filter_map(|e| async { Ok(e.right().map(|c| c.version)) })
+                .try_collect::<Vec<String>>()
+                .await?;
 
-            // TODO: only loaders for recent versions? For mods that have moved from forge to fabric
```

Comment:
This is still an issue, although it may not be the best place for this.  We may want to move these to github issues or a centralized list.